compat/themify-rules-configuration.json

```diff
@@ -0,0 +1,33 @@
+{
+  "rules": [
+    { "class": "PassThroughRule", "selector": "div" },
+    {
+      "class": "InteractiveRule",
+      "selector": "div.video-wrap",
+      "properties": {
+        "interactive.iframe": {
```

Comment:
I think the correct property is `interactive.url`, as with `interactive.iframe` the markup I get is:
```html
<iframe class="no-margin">https://www.youtube.com/embed/waM20ewLj34?feature=oembed</iframe>
```
However, if I only change the property to `interactive.url`, the resulting markup does NOT contain the iframe element (not sure if this is an existing bug or the result of missing required properties.
The following properties worked well for me:
```javascript
"properties": {
        "interactive.url": {
          "type": "string",
          "selector": "iframe",
          "attribute": "src"
        },
        "interactive.height": {
          "type": "int",
          "selector": "iframe",
          "attribute": "height"
        },
        "interactive.width": {
          "type": "int",
          "selector": "iframe",
          "attribute": "width"
        }
      }
```